private-key-compromise/src/agent.ts

```diff
@@ -5,30 +5,43 @@ import {
   updateRecord,
   TIME_PERIOD,
   AlertedAddress,
+  UpdatedAlertedAddress,
   QueuedAddress,
   ERC20_TRANSFER_FUNCTION,
   deepMerge,
   MAX_OBJECT_SIZE,
+  ONE_DAY_IN_BLOCKS,
+  SIX_HOURS_IN_BLOCKS,
+  FIVE_MINS_IN_SECS,
+  ONE_WEEK_IN_SECS,
 } from "./utils";
 import { NetworkManager } from "forta-agent-tools";
-import { createFinding, createDelayedFinding } from "./findings";
+import { createFinding, createHighVictimAmountFinding, createDelayedFinding } from "./findings";
 import calculateAlertRate, { ScanCountType } from "bot-alert-rate";
 import BalanceFetcher from "./balance.fetcher";
 import DataFetcher from "./data.fetcher";
 import ContractFetcher from "./contract.fetcher";
 import MarketCapFetcher from "./marketcap.fetcher";
 import PriceFetcher from "./price.fetcher";
 import { PersistenceHelper } from "./persistence.helper";
-import CONFIG, { priceThreshold } from "./bot.config";
+import CONFIG, { priceThreshold, MIN_TRANSFER_AMOUNT_THRESHOLD, HIGH_TRANSFER_AMOUNT_THRESHOLD } from "./bot.config";
 import { getSecrets, ApiKeys, BlockExplorerApiKeys } from "./storage";
 
 import fetch from "node-fetch";
 
 const DATABASE_URL = "https://research.forta.network/database/bot/";
+// Prod keys
+// const DATABASE_OBJECT_KEYS = {
+//   transfersKey: "nm-pk-comp-bot-key-1",
+//   alertedAddressesKey: "nm-pk-comp-bot-alerted-addresses-key-1",
+//   queuedAddressesKey: "nm-pk-comp-bot-queued-addresses-key-1",
+// };
```

Comment:
```suggestion
const getDatabaseObjectKeys = () => {
  const isProd = process.env.NODE_ENV === "production";

  return {
    transfersKey: isProd ? "nm-pk-comp-bot-key-1" : "nm-pk-comp-bot-key-1-test",
    alertedAddressesKey: isProd
      ? "nm-pk-comp-bot-alerted-addresses-key-1"
      : "nm-pk-comp-bot-alerted-addresses-key-1-test",
    queuedAddressesKey: isProd ? "nm-pk-comp-bot-queued-addresses-key-1" : "nm-pk-comp-bot-queued-addresses-key-1-test",
  };
}

const DATABASE_OBJECT_KEYS = getDatabaseObjectKeys();
```
maybe we could use `env` variable to figure out whether it's prod or test

Reply:
It wasn't meant to be for testing, but more for the beta deployment. Similar to how you or Vasilis have been adding a random character at the end of some keys to not use the prod ones.

So not sure that we need to have different sets, one for prod and one for tests.

Reply:
I see. if it's going to continue to be the same in prod, then it can stay like this of course 👍